.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -142,17 +138,24 @@ jobs:
       run: |
         cmake --build $GITHUB_WORKSPACE/build --target check-xptifw
     - name: Install
-      if: ${{ steps.parameters.build_upload_artifact }}
-      run: cmake --build $GITHUB_WORKSPACE/build --target install
+      # TODO replace utility installation with a single CMake target
```

Comment:
What is the problem with `install` target?

Reply:
Package size is over 1.5 gigs, and most of that stuff is unused. The new set of install targets only packages what's needed for running tests.